app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -383,7 +387,7 @@ PortLoop:
 	}
 
 	if port == nil {
-		return ingressbe.ServicePort{}, fmt.Errorf("could not find matching nodeport for backend %v and service %s/%s. Looking for port %v in %v", be, namespace, be.ServiceName, be.ServicePort, svc.Spec.Ports)
+		return ingressbe.ServicePort{}, fmt.Errorf("Could not find matching nodeport for backend %+v and service %s/%s. Looking for port %+v in %v", be, namespace, be.ServiceName, be.ServicePort, svc.Spec.Ports)
```

Comment:
Libraries are supposed to return errors starting with smaller case so that the caller can prepend context